site/src/api/queries/insights.ts

```diff
@@ -20,3 +20,10 @@ export const insightsUserActivity = (params: InsightsParams) => {
 		queryFn: () => API.getInsightsUserActivity(params),
 	};
 };
+
+export const userStatusCountsOverTime = () => {
+	return {
+		queryKey: ["userStatusCountsOverTime"],
```

Comment:
Following the pattern of the other queries, the `queryKey` should be `["insights", "userStatusCountsOverTime"]`.